prac_06/car.py

```diff
@@ -4,14 +4,18 @@
 class Car:
     """Represent a Car object."""
 
-    def __init__(self, fuel=0):
+    def __init__(self, name="", fuel=0):
         """Initialise a Car instance.
 
         fuel: float, one unit of fuel drives one kilometre
         """
+        self.name = name
         self.fuel = fuel
         self._odometer = 0
 
```

Comment:
Seems like you are missing a 'drive' function...

Reply:
No, it is there? line 19